src/main/java/at/hannibal2/skyhanni/data/EntityMovementData.kt

```diff
@@ -41,6 +46,17 @@ class EntityMovementData {
         }
     }
 
+    private val warpingPattern by RepoPattern.pattern("warping", "§7Warping...")
```

Comment:
There are multiple warp messages that are possible that this will miss, see: [ChatFilter.kt#L71](https://github.com/hannibal002/SkyHanni/blob/beta/src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt#L71)

I also dont really understand the point of a warp event when the `LorenzWorldChangeEvent` exists. If its because hypixel calls it 3 times then do what I did in [DiscordRpcManager.kt](https://github.com/hannibal002/SkyHanni/blob/beta/src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordRPCManager.kt#L177) and cache the time when it will run the runnable and then check if its in the future and not try to run a delayed task again.

Reply:
A WorldChange mustn't be the case here, this also triggers when you warp locally in the same lobby. The other messages shouldn't matter since they only happen if you warp to an island that your not on. Sure the event isn't perfect since it also triggers when world changing but it works.